src/payment/strategies/zip/zip-payment-strategy.ts

```diff
@@ -80,6 +82,10 @@ export default class ZipPaymentStrategy implements PaymentStrategy {
                                     return resolve(checkoutId);
                                 }
 
+                                if (state === ZipModalEvent.CheckoutDeclined) {
+                                    return reject(new StandardError(this._zipDeclinedMessage));
```

Comment:
There's no need for a constant here as this is the only place were it's been used.